static/src/stylesheets/module/content/tones/_tone-media.scss

```diff
@@ -58,6 +58,7 @@
         }
     }
 
+    &:not(.content--advertisement-feature)
     .brandbadge__logo {
```

Comment:
Can put this on the same line as it's part of the same selector:

`&:not(.content--advertisement-feature) .brandbadge__logo {`